subsys/bluetooth/host/Kconfig

```diff
@@ -173,6 +173,16 @@ config BT_SETTINGS
 	  instead of each subsystem doing it independently.
 
 if BT_SETTINGS
+config BT_SETTINGS_CCC_LOAD_ON_DEMAND
```

Comment:
@fnde-ot if it's not too late for the feature freeze today, can we rename this to:

`BT_SETTINGS_CCC_LAZY_LOADING` ?

Reply:
I don't mind if you prefer. Just to clarify, would "lazy loading" mean that we load everything at once (lazy because we do it only once at init time) or to load when a device connects (on demand)?

Reply:
Just pushed this change!